pkg/qemu/devices.go

```diff
@@ -81,6 +97,99 @@ func (rod ReadOnlyDirectory) Cmdline() []string {
 		"-device", "ide-drive,drive=tmpdir,bus=ahci.0",
 	}
 }
+func (ReadOnlyDirectory) KArgs() []string { return nil }
+func (rod ReadOnlyDirectory) Cleanup(t ErrorLogger) {
```

Comment:
Hm, this seems more like a testing.TB than ErrorLogger to me, wdyt?

Reply:
fixed